Tests/TuistKitTests/Services/BuildServiceTests.swift

```diff
@@ -71,7 +71,7 @@ final class BuildServiceTests: TuistUnitTestCase {
             XCTAssertEqual(_path, path)
             return workspacePath
         }
-        buildgraphInspector.buildArgumentsStub = { _target, _ in
+        buildgraphInspector.buildArgumentsStub = { _target, _, _ in
```

Comment:
Rather than ignoring the new argument, I'd keep it and assert that the skip tests value is the expected one.

Reply:
I'd do that with every test case inside this class.